web/src/features/charts/elements/AreaGraph.tsx

```diff
@@ -20,7 +20,7 @@ import GraphHoverLine from './GraphHoverline';
 import ValueAxis from './ValueAxis';
 
 const X_AXIS_HEIGHT = 20;
-const Y_AXIS_WIDTH = 20;
+const Y_AXIS_WIDTH = 26;
```

Comment:
Just an idea here, but not sure if it should be in this PR or just a future improvement but it would be nice to calculate the actual with needed instead of having to use a constant.

Reply:
Yes that would be nice! In the design review meeting we discussed units which will also affect how much space is needed, so I think it is a future thing.